Dockerfile

```diff
@@ -19,11 +25,12 @@ COPY ./conf/LocalSettings.php /var/www/html/LocalSettings.php
 # Composer
 RUN apt update
 RUN apt install zip unzip
-COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer /usr/bin/composer /usr/local/bin/composer
```

Comment:
BTW You can also download the composer directly in the image with:
`RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer`

Reply:
That's also one possible way of doing this, but according to the composer documentation the better way is using the docker image provided by them.